github_action_resources/entrypoint.sh

```diff
@@ -101,7 +103,10 @@ echo "BC_REPOSITORY_URL="${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}""
 
 echo "running checkov on directory: $1"
 
-if [ -n "$API_KEY_VARIABLE" ]; then
+if [ -n "$API_KEY_VARIABLE" ] && [ -n "$DOCKER_IMAGE"]; then
+  echo "checkov --bc-api-key XXXXXXXXX-XXX-XXXXX --branch $GIT_BRANCH --repo-id $GITHUB_REPOSITORY $DOCKER_IMAGE_FLAG $DOCKERFILE_PATH_FLAG" #Unsure what other flags should work
```

Comment:
hard coded xxx api key?

Reply:
That's just for the echo. I copied what the previous one did (see below). I'll update both.